leptos/src/for_loop.rs

```diff
@@ -24,19 +25,7 @@ use std::hash::Hash;
 ///     <div>
 ///       <For
 ///         // a function that returns the items we're iterating over; a signal is fine
-///         each=move || counters.get()
-///         // a unique key for each item
-///         key=|counter| counter.id
-///         // renders each item to a view
-///         children=move |counter: Counter| {
-///           view! {
-///             <button>"Value: " {move || counter.count.get()}</button>
-///           }
-///         }
-///       />
```

Comment:
This was repeated twice. I'm assuming that was unintentional. Let me know if you want to keep it.